cmd/kube-scheduler/app/options/insecure_serving.go

```diff
@@ -53,17 +53,17 @@ func (o *CombinedInsecureServingOptions) AddFlags(fs *pflag.FlagSet) {
 }
 
 func (o *CombinedInsecureServingOptions) applyTo(c *schedulerappconfig.Config, componentConfig *kubeschedulerconfig.KubeSchedulerConfiguration) error {
-	if err := updateAddressFromDeprecatedInsecureServingOptions(&componentConfig.HealthzBindAddress, o.Healthz); err != nil {
+	if err := updateAddressFromDeprecatedInsecureServingOptions(componentConfig.HealthzBindAddress, o.Healthz); err != nil {
```

Comment:
Most of these methods assume that the passed-in pointer is non-nil. Using a value that can now be nil adds the risk of a NPE, for little apparent benefit.